src/com/sun/jna/Library.java

```diff
@@ -271,4 +271,5 @@ public Object invoke(Object proxy, Method method, Object[] inArgs)
             }
         }
     }
+
```

Comment:
I love whitespace! ;)

Reply:
I'm deeply sorry, but it should not have been there, so it has to go again :grin: